docs/source/usage/cli.rst

```diff
@@ -0,0 +1,156 @@
+Command-line Interface
+=====================
+Metasyn provides a command-line interface (CLI) for accessing core functionality without needing to write any Python code.
+
+The CLI currently has two subcommands. The ``synthesize`` subcommand, which allows you to **generate synthetic data** from any ``GMF file``, and the ``schema`` subcommand, which allows you to **create validation schemas** for a :obj:`MetaFrame<metasyn.metaframe.MetaFrame>`.
+
+
+At any point, you can also use the help command to get more information about the CLI and its subcommands:
+
+.. code-block:: console
+
+   metasyn --help
+
+
+Accessing the CLI
+-----------------
+If you have installed the main metasyn package, the CLI should be available automatically. You can find instructions on how to install metasyn in the :doc:`installation` section of the documentation.
+
+Alternatively, the CLI can be accessed through a Docker container, allowing you to run metasyn in an isolated environment without installing the package on your system. This can be useful, for example, when trying out metasyn without affecting your existing Python environment.
+
+Here's how you can use Docker to access Metasyn's CLI:
+
+1. **Install Docker:** If Docker isn't already set up on your machine, please follow the instructions on `Docker's official website <https://docs.docker.com/get-docker/>`_.
+
+2. **Pull the metasyn Docker Image:** After successfully installing Docker, you can download the metasyn Docker image from Docker Hub using the following command.
+
+   .. code-block:: console
+
+      docker pull sodateam/metasyn
+
+3. **Run metasyn's CLI via Docker:** Once the Docker image is downloaded, you can use the following command to run a metasyn CLI command within a Docker container (in this case ``--help``), and simultaneously set the working directory (which we denote as `wd` in this case).
+
+   .. tab:: Windows
+
+      .. code-block:: console
+
+         docker run -v %cd%:/wd sodateam/metasyn --help
+
+   .. tab:: Unix or MacOS:
+
+      .. code-block:: console
+
+         docker run -v $(pwd):/wd sodateam/metasyn --help
+
+
+The Metasyn CLI should now be up and running within the Docker container and ready for use!
+
+.. note:: 
+   You can also specify which metasyn version to use in docker, by adding a tag to the docker image. For example, to use version 0.5, you can use the following command:
```

Comment:
This might be confusing with the name change, since the first metasyn version is 0.6.0.

Reply:
I would just use 0.6 instead.

Reply:
Yeah the idea was to reflect the fact that older versions can be installed, but you're right, due to there not being an older release, 0.6 is the smarter option.